.travis.yml

```diff
@@ -8,7 +8,6 @@ cache:
     - $HOME/.cache
 
 before_install:
-  - if [[ `npm -v` != 5* ]]; then npm install -g npm@5; fi
   - npm -v
```

Comment:
Since you're not installing a newer version of `npm` than the default, you can remove this line, as the version of `npm` is already printed a few lines above (in line 458 in the screenshot below) in the output. (This line results in the output on line 463 in the screenshot.)

<img width="819" alt="screen shot 2019-02-09 at 5 05 42 pm" src="https://user-images.githubusercontent.com/718899/52528195-11135000-2c8d-11e9-84f9-233184b4ed9b.png">

Reply:
👍